adafruit_bitmap_font/bdf.py

```diff
@@ -22,8 +22,15 @@
 
 """
 
+try:
+    from typing import Union, Optional, Tuple
+except ImportError:
+    pass
+
 import gc
+from io import FileIO
```

Comment:
For reference, this is so we don't import them within a microcontroller environment, and only in a development environment.  This is okay since the CircuitPython doesn't actually pay attention to the type annotations, so it's okay for them not to be defined at runtime.  Overall, this saves us some RAM by only importing what is needed!

To prevent them from being imported, we use the above `try`/`except` block with the first contained statement being an import being from the `typing` library, which doesn't exist within CircuitPython.  Thus, it will fail on the microcontroller and skip importing the rest of the imports inside.

Hopefully that provides some context!

Reply:
Updated!